### ISSUE TITLE
NINTENDONT "gcn_md5.txt NOT FOUND"

### ISSUE THREAD

**Issue by meswee** [FIX94/Nintendont#911]
I've been using Nintendont for a year now and it has been working fine up until now. The weirdest issue I've ever come across. 

So I boot up Nintendont, select USB (I use an official gamecube controller btw), I start looking for a game and then suddenly it brings up the settings, which you can only access by hitting 'X', and it locks me on that screen with an error message on the bottom that says: "gcn_md5.txt NOT FOUND". I can't even turn it off with the power button so I have to unplug the Wii. I've been looking around for threads with people with the same issue and nothing has helped. Then something happened afterwards; the weirdest thing ever...

 I discover that if I unplug my GC controller, I can use my official Wiimote to navigate around Nintendont just fine. So I boot up a game, I plug the GC controller again, I try to use the 'X' button and it just doesn't work. I've never dropped it and I've had it for over 2 years and it just stopped working. The 'Y' button had a different functionality than the one it was meant to have as well. I restart the Wii and boot up a GC game without Nintendont to see if the problem still persists.

I hit the 'X' button and it still doesn't work; however, the 'Y' button worked just fine. I unplug and plug the GC controller again, and suddenly, the action that was mapped to the 'X' button gets buffered without me pressing the X button. Turns out that the moment the GC controller gets plugged in, it buffers the X button action. So as you can see, a lot of weird things are going on related with the X button action. I sure hope my GC controller didn't get permanently damaged.

I really don't know what to do. I even reinstalled Nintendont and nothing. I'd really appreciate it if someone could help me out with this.

**Comment by carnage702** [comment]
Take a picture of your nintendont settings

**Comment by meswee** [comment]
Hey carnage702, I appreciate you coming in to help me out! But it seems like I solved my problem... I think? 
What I did was disconnect everything from my wii and left it like that for around 4 hours. When I turned it on to send you a picture of the settings, everything started working fine. My controller is back to normal as well. I don't know what could've started it.
I read somewhere that nintendont doesnt take 3rd party things well so I disconnected my 3rd party memory card and the other stuff for good measure. And well... It's working now.
